src/super_gradients/recipes/coco_ssd_lite_mobilenet_v2.yaml

```diff
@@ -0,0 +1,131 @@
+defaults:
+  - training_hyperparams: default_train_params
+  - dataset_params: coco_detection_dataset_params
+  - arch_params: default_arch_params
+  - checkpoint_params: default_checkpoint_params
```

Comment:
Don't we start from a pretrained backbone anymore?
If we do please also add it to the model zoo, and add whatever else needed in the yaml file for it to be loaded.

In case we do- also add support for normalising by imagenet mean and std in coco dataset interface.

Reply:
Extensive experiments show that there is no improvement from using ImageNet pre-trained backbone for COCO detection, or we aren't doing it right. 
As for my run with pre-training and image normalization, it landed 0.4 mAP lower. 
We can still add `pretrained: False` to the yaml and make sure True would work correctly. WDYT?

Reply:
If we didn't experience any improvement then sure, lets leave it.